cookiecutter.json

```diff
@@ -15,6 +15,7 @@
     "GNU General Public License v3",
     "Other"
   ],
+  "include_src_folder": "yes",
```

Comment:
I would calle it "project_layout" with option "src" or "flat" feel free to keep src as the default one (1)